src/panel/context/Timeline.tsx

```diff
@@ -1,19 +1,23 @@
 import React, {
   createContext,
+  Dispatch,
   useRef,
   useState,
   useCallback,
   useMemo,
   FC,
+  SetStateAction,
   useContext,
   useEffect
 } from "react";
 import { scaleLinear } from "d3-scale";
-import { ParsedEvent } from "../types";
+import { PresentedEvent } from "../types";
 import { DevtoolsContext } from "./Devtools";
 
 interface TimelineContextValue {
   events: Record<string, any>;
+  selectedEvent: PresentedEvent | null;
```

Comment:
Super nit picky - trying to stick to undefined where possible

```suggestion
  selectedEvent?: PresentedEvent;
```